spectrum/autism_prevalence_map/views.py

```diff
@@ -83,16 +83,7 @@ def about(request):
 	"""
 	  About page
 	"""
-	if os.environ["DJANGO_ALLOWED_HOSTS"] == 'prevalence-staging.spectrumnews.org' :
```

Comment:
I was thinking maybe we want to preserve this for the redesign, but we should get rid of the dependence on an external site's stylesheet by then anyway